### ISSUE TITLE
Jaws does not read aria-describedby of table if table column headers are sortable with tabindex

### ISSUE THREAD

**Issue by hmilyliu8** [FreedomScientific/VFO-standards-support#397]
### Summary
Jasws ignore aria-descriedby of sortable table.
> Brief description of the issue. Please list any specific steps.
> 
> Example:
1. create a table with role= grid and set aria-label and aria-describedby attributes
2. make table header focus-able by adding tab-index=0
Use T to go to table
It will start from the first column header and read column name, table aria-label but no aria-describedby

Remove all tab-index and try again, now it reads everything.

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number
Jaws 2020.2003.13.ILM

 
#### Browser and version:
Chrome Version 83.0.4103.116

**Comment by JAWS-test** [comment]
In my opinion, descriptions (title, aria-describedby, etc.) are only output when navigating with the Tab key and not with the virtual cursor, which seems to be a JAWS default setting that is not correct but has nothing to do with grid, tabindex, etc. However, to check your specific example, you should link to a source code example or insert it here

**Comment by simlin** [comment]
Hi,

is there any update on this?
We stumbled over the same problem. Jaws **2025** does not announce the aria-describedby on grid level.
I just took the first example from https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/data-grids/ and added an aria-describedby (see https://codefile.io/f/PbSIhF2r1o).

Thx
